### DISCUSSION TITLE
RAG app in python with Redis as vector store

### DISCUSSION THREAD

**Discussion by sjadlakha** [microsoft/semantic-kernel#5979] [Q&A] [answered]
Hi,

I see that in the docs, the support for Redis in python as vector store is a wip.
Q1: What is the expected timeline for it's release?
Q2: Till then, is there any sample or custom implementation that I can build upon for redis connection?

**Comment by CawaMS** [comment, marked as answer]
The Python memory store support for Python is there already. It will be updated again post //BUILD. https://github.com/microsoft/semantic-kernel/tree/main/python/semantic_kernel/connectors/memory/redis
Adding @alliscode who created this support.

**Reply by engt21** [reply]
@CawaMS  @alliscode 
I'm not sure if these issues were going to be addressed post MSFT Build, but within my enterprise we've discovered several issues/missing features that we locally modified and got to work for the redis memory connector. I've raised them as issues, and pending approval can raise PRs with the fixes that we are using. 
https://github.com/microsoft/semantic-kernel/issues/6084
https://github.com/microsoft/semantic-kernel/issues/6088
